From 67fe56a61d1383daaa5513d7e09a04fa6a8642c0 Mon Sep 17 00:00:00 2001
From: Denys Vlasenko <dvlasenk@redhat.com>
Date: Tue, 26 Feb 2013 16:53:36 +0100
Subject: [ABRT EL6 PATCH 137/197] abrtd: Remove "Corrupted or bad directory"
 words from the message

In many cases, directory isn't "corrupted or bad".
It may be just a crash in unpackaged app.

Signed-off-by: Denys Vlasenko <dvlasenk@redhat.com>
---
 src/daemon/abrtd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/daemon/abrtd.c b/src/daemon/abrtd.c
index f54cd61..8265363 100644
--- a/src/daemon/abrtd.c
+++ b/src/daemon/abrtd.c
@@ -463,7 +463,7 @@ static gboolean handle_event_output_cb(GIOChannel *gio, GIOCondition condition,
 
 
  delete_bad_dir:
-    log("Corrupted or bad directory '%s', deleting", state->dirname);
+    log("Deleting problem directory '%s'", state->dirname);
     delete_dump_dir(state->dirname);
 
  ret:
-- 
1.8.2.1

